WordPress/Classes/ViewRelated/Plugins/PluginViewModel.swift

```diff
@@ -372,6 +391,20 @@ class PluginViewModel: Observable {
             ])
     }
 
+    private func confirmRegisterDomainAlert() -> UIAlertController {
+        let title = NSLocalizedString("Install Plugin", comment: "Install Plugin dialog title.")
+        let message = NSLocalizedString("To install plugins you need to have a custom domain associated with your site", comment: "Install Plugin dialog text.")
+        let registerDomainActionTitle = NSLocalizedString("Register domain", comment: "Install Plugin dialog register domain button text")
+
+        let alertController = UIAlertController(title: title, message: message, preferredStyle: .alert)
+
+        alertController.addCancelActionWithTitle(NSLocalizedString("Cancel", comment: "Cancel removing a plugin"))
+        alertController.addDefaultActionWithTitle(registerDomainActionTitle) { [weak self] (action) in
```

Comment:
Per the design, 'Register domain' should be in bold and 'Cancel' should not.

Also, the design shows the Cancel button on top. But I actually like it on the bottom as you have it. 👍

Reply:
Done